TypeCobol.Test/Utils/Comparator.cs

```diff
@@ -505,6 +505,13 @@ public override bool Visit(SelectStatement selectStatement)
                 return true;
             }
 
+            public override bool Visit(RollbackStatement rollbackStatement)
+            {
+                _writer.WriteLine("line " + rollbackStatement.Line + ": RollbackStatement");
```

Comment:
Now that i read it, we can use `nameof` also for 'RollbackStatement', so

```csharp
_writer.WriteLine($"line {rollbackStatement.Line}: {nameof(RollbackStatement)}");
```

Please change also the other methods for SelectStatement and CommitStatement